frontend/pages/SoftwarePage/SoftwareAddPage/SoftwareCustomPackage/SoftwareCustomPackage.tsx

```diff
@@ -150,32 +149,7 @@ const SoftwareCustomPackage = ({
         </>
       );
     } catch (e) {
-      const isTimeout =
-        isAxiosError(e) &&
-        (e.response?.status === 504 || e.response?.status === 408);
-      const reason = getErrorReason(e);
-
-      if (isTimeout) {
-        renderFlash(
-          "error",
-          `Couldn’t upload. Request timeout. Please make sure your server and load balancer timeout is long enough.`
-        );
-      } else if (reason.includes("Fleet couldn't read the version from")) {
```

Comment:
Is there a reason this was removed? I added it back, but lmk

Reply:
I think I mistakenly deleted this. Thanks for catching

Reply:
@ghernandez345 thanks for cleaning up the error code!!!